.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+PR title should follow the pattern: <keyword>(optional scope): <description>. If this is a feat/fix, add the technical writer as a reviewer to the PR.
```

Comment:
Thanks Flora. Some final thoughts 

- Can we make this a comment so that if it isn't removed it doesn't show up in rendered format?
- Can we replace `keyword` by `type` and also then say `If the type is "feat" or "fix"`?
- What do you think about adding a link to https://www.conventionalcommits.org/en/v1.0.0/#summary for reference of types and format ?

Reply:
@mohanraj-r Sounds great! I'll take those two suggestions. Updating the doc now.